components/massmint/OverviewTable.vue

```diff
@@ -5,21 +5,191 @@
         {{ $t('massmint.overviewTable') }}
       </div>
       <template #content>
-        <div>Overview table goes here</div>
+        <div class="limit-height">
+          <div class="columns is-mobile m-0 px-4 py-3 border-bottom">
+            <div class="column has-text-grey">#</div>
+            <div class="column has-text-grey">{{ $t('massmint.image') }}</div>
+            <div class="column has-text-grey">{{ $t('massmint.name') }}</div>
+            <div class="column is-3 has-text-grey">
+              {{ $t('massmint.description') }}
+            </div>
+            <div class="column has-text-grey">{{ $t('massmint.price') }}</div>
+            <div class="column has-text-grey">{{ $t('massmint.status') }}</div>
+            <div class="column has-text-grey">
+              {{ $t('massmint.operation') }}
+            </div>
+          </div>
+          <div
+            v-for="nft in displayedNFTS"
+            :key="nft.id"
+            class="columns is-mobile border-bottom m-0 py-2 px-4">
+            <div class="column is-flex is-align-items-center has-text-grey">
+              {{ nft.id }}
+            </div>
+            <div class="column is-flex is-align-items-center">
+              <img :src="nft.imageUrl" class="image is-48x48 border cover" />
+            </div>
+            <div class="column is-flex is-align-items-center">
+              <div
+                class="is-clickable"
+                :class="{
+                  'has-text-k-red': !nft.name,
+                  'has-text-grey': nft.name,
+                }"
```

Comment:
is it better to use conditional operator like `nft.name ? 'has-text-grey' : 'has-text-k-red'`?

Reply:
that would require creating a new  computed ref, which i didn't want to do.
https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes

Reply:
> that would require creating a new computed ref

i'm not clear, why? there's an example in the doc you provided
> If you would like to also toggle a class in the list conditionally, you can do it with a ternary expression:
`<div :class="[isActive ? activeClass : '', errorClass]"></div>`

but nvm, it's just a small suggestion, you can do it your way :)